### ISSUE TITLE
Difference between proposed inception resnet v2 and tensorflow's repo inception resnet v2 model

### ISSUE THREAD

**Issue by christk1** [davidsandberg/facenet#1062]
i tried to train my dataset with the tensorflow's repo inception resnet v2 model but it yields different results than that from your inception model.. I am not familiar with slim package as i started learning deep learning with tf.keras...Can you explain me what makes your model different and yields "correct" results? @davidsandberg

**Comment by christk1** [comment]
i resolved it, batchnorm wasn't in the arg_scope

**Comment by blueprintparadise** [comment]
Can you share your code. I am trying to replace the inceptionresnetv2 with the keras version but I keep getting errors.
